addons/account/views/report_invoice.xml

```diff
@@ -355,8 +355,10 @@
                     <t t-else="">
                         <td>
                             <span t-out="amount_by_group['tax_group_name']">Tax 15%</span>
-                             on 
-                            <span class="text-nowrap" t-out="amount_by_group['formatted_tax_group_base_amount']">27.00</span>
+                            <span t-if="amount_by_group['display_formatted_tax_group_base_amount']">
```

Comment:
what about:
```suggestion
                            <span t-if="amount_by_group['formatted_tax_group_base_amount'] is not None">
```
instead of adding another value in a way too big dictionary?